Makefile

```diff
@@ -97,14 +97,9 @@ testprereqs: $(KUBEBUILDER) $(KUSTOMIZE)
 test: testprereqs fmt lint ## Run tests
 	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./api/... ./controllers/... ./baremetal/... -coverprofile ./cover.out
 
-.PHONY: test-integration
-test-integration: ## Run integration tests
-	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v -tags=integration ./test/integration/...
-
```

Comment:
We should also clean up this target from Jenkins Job Builder of `job_capi_bm_v1a5_integration_tests_ubuntu.yml` ?

Reply:
Sorry, I mismatched with another job. Please ignore it.